kedro-telemetry/tests/test_masking.py

```diff
@@ -47,9 +47,10 @@ def fake_metadata(fake_root_dir):
         PACKAGE_NAME,
         "CLI Tools Testing Project",
         fake_root_dir / REPO_NAME,
-        kedro_version,
         fake_root_dir / REPO_NAME / "src",
         kedro_version,
+        [],
+        "No",
```

Comment:
Could we use named arguments here? It's getting a bit hard to understand what all these inputs are..

Reply:
Thank you, I fully agree, fixed